test/t1001-generate-template.t

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+test_description='Test generate template'
+
+. ./sharness.sh
+
+test_expect_failure CAT,ECHO,HEAD,JQ 'Test ocitools generate --template with an empty template' "
+	echo '{}' >template &&
+	ocitools generate --template template | jq . >actual &&
+	cat <<-EOF >expected &&
+		{
+		  \"ociVersion\": \"1.0.0-rc1\",
```

Comment:
On Wed, Aug 03, 2016 at 04:06:28PM -0700, Mrunal Patel wrote:
> > @@ -0,0 +1,59 @@
> > +#!/bin/sh
> > +
> > +test_description='Test generate template'
> > +
> > +. ./sharness.sh
> > +
> > +test_expect_failure CAT,ECHO,HEAD,JQ 'Test ocitools generate --template with an empty template' "
> > +	echo '{}' >template &&
> > +	ocitools generate --template template | jq . >actual &&
> > +	cat <<-EOF >expected &&
> > +		{
> > +		  \"ociVersion\": \"1.0.0-rc1\",
> 
> I don't like the escaping that we do here :/

There are only so many quotes we can use ;).  Would you rather me wrap
the test in single quotes and use double quotes inside it?  That makes
using variables in the test tricky, but we don't do any of that yet.